hpOneView/resources/servers/server_profiles.py

```diff
@@ -403,20 +408,19 @@ def get_available_targets(self, **kwargs):
         profile.
 
         Args:
-           enclosureGroupUri (str):
-               The URI of the enclosure group associated with the resource.
-           serverHardwareTypeUri (str):
-               The URI of the server hardware type associated with the resource.
-           profileUri (str):
-               The URI of the server profile associated with the resource.
+           enclosureGroupUri (str): The URI of the enclosure group associated with the resource.
+           serverHardwareTypeUri (str): The URI of the server hardware type associated with the resource.
+           profileUri (str): The URI of the server profile associated with the resource.
+           scopeUris (str): An expression to restrict the resources returned according to
+               the scopes to which they are assigned.
 
         Returns:
             list: List of available servers and bays.
         """
         uri = self.__build_uri_with_query_string(kwargs, '/available-targets')
         return self._client.get(uri)
 
-    def __build_uri_with_query_string(self, kwargs, sufix_path, id_or_uri=None):
+    def __build_uri_with_query_string(self, kwargs, sufix_path='', id_or_uri=None):
```

Comment:
is this required? in this file, we are always appending the path whenever we are calling this method.

Reply:
Yes, This is required.
Added default value for sufix_path as we have some methods where we have to generateURI without sufix_path.
Ex: create method.

Reply:
ok. i missed this one. LGTM 👍